package.json

```diff
@@ -11,7 +11,9 @@
     "clean": "lerna clean && lerna run clean",
     "format": "prettier --write \"packages/**/*.{ts,js,tsx,jsx,json}\"",
     "lint": "eslint packages/ --ext .js,.ts,.jsx,.tsx",
-    "test:lhci": "lhci autorun --collect.url=/index.html"
+    "test:lhci": "lhci autorun --collect.url=/index.html",
```

Comment:
I think we can remove this `test:lhci`. This is from the time the stores were on this repo